src/gui/static/src/environments/environment.prod.ts

```diff
@@ -1,5 +1,5 @@
 export const environment = {
-  nodeUrl: 'http://127.0.0.1:6420/',
+  nodeUrl: '../' ,
```

Comment:
Does this always work ? Shouldn't it be an absolute URL ?

Reply:
@olemis  absolute urls are set when you are only going to point to a standard address, but in this case they change the addresses a lot, that is, ip: random (port) is declared in this way so that it reads the api from the address that continues ".. / ".